.github/workflows/cre-system-tests.yaml

```
@@ -137,15 +139,39 @@ jobs:
       # TODO
       # Add Flakeguard support
 
-      - name: Run CRE system tests
+      - name: Enable cron capability
         shell: bash
-        working-directory: system-tests/tests
+        working-directory: core/scripts/cre/environment/configs
+        run: |
+          echo "[extra_capabilities]" >> "ci-config.toml"
+          echo 'cron_capability_binary_path = "${{ github.workspace }}/system-tests/tests/smoke/cre/cron"' >> "ci-config.toml"
+
+          cat "ci-config.toml"
```

```suggestion
        working-directory: core/scripts/cre/environment/configs
        run: |
          echo "[extra_capabilities]" >> "ci-config.toml"
          echo 'cron_capability_binary_path = "${GITHUB_WORKSPACE}/system-tests/tests/smoke/cre/cron"' >> "ci-config.toml"

          cat "ci-config.toml"
```

See: https://docs.github.com/en/actions/reference/workflows-and-actions/variables -- if they have an env var available, we should prefer it so everything in the `run: ` value is valid bash.